src/screens/helping/services.jsx

```diff
@@ -0,0 +1,162 @@
+import React from 'react'
+import { connectModule } from 'redux-modules'
+import { compose, withStateHandlers } from 'recompose'
+import servicesModule from '../../modules/services'
+import Layout from '../../components/Layout'
+import { Container, HeaderContainer } from '../../components/atoms'
+import { Input, Radio, Checkbox, Button } from 'antd'
+import styled from 'styled-components'
+const RadioGroup = Radio.Group
+const CheckboxGroup = Checkbox.Group
+const { TextArea } = Input
+
+const RequiredIndicator = styled.em`
+  color: red;
+`
+
+const Label = styled.div`
+  font-size: 18px;
+  text-transform: uppercase;
+  padding: 15px 0;
+`
+
+const StackContainer = styled.div`
+  margin: 0 0;
+`
+
+const FormSection = styled.div`
+  padding: 30px 0;
```

Comment:
Since a lot of these form elements are being copied from the other form.

It would be killer if you put all the form elements in one file (similar to `components/atoms`) so that we can share styles easily.

Reply:
Done, created formAtoms.jsx (not sure about the name, or if it should be camelCase)